CRM/Financial/BAO/ExportFormat/IIF.php

```diff
@@ -351,7 +351,7 @@ public static function format($s, $type = 'string') {
     $s1 = str_replace(self::$SEPARATOR, '\t', $s);
     switch ($type) {
       case 'date':
-        $sout = date('Y/m/d', strtotime($s1));
+        $sout = CRM_Utils_Date::customFormat(date('Ymd', strtotime($s1)));
```

Comment:
![image](https://cloud.githubusercontent.com/assets/5624277/15447023/5aff5450-1f4f-11e6-8c69-04e84f206c9f.png)
![image](https://cloud.githubusercontent.com/assets/5624277/15447027/9f3ea918-1f4f-11e6-81d5-e5823b0fd04d.png)

@JoeMurray Hi Joe. The suggested change does take the date format from (Localization > Date Formats > Date Display > Date Format: Complete Date) and display accordingly in IIF file. But adding a new field to Localization does sound better. I will work on this and  update the PR. :)